### PR TITLE
Add FilterSearch Component to README

### PR DESCRIPTION
**Pull request by Delerme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,6 +241,22 @@ ANSWERS.addComponent('SearchBar', {
 })
 ```
 
+## FilterSearch Component
+
+The FilterSearch component provides a text input box for users to type a query and select a preset matching filter. When a filter is selected, a vertical search is performed.
+
```
Comment (b1lly) [inline README.md:247]: Do you want to make a note about how it works if you add `n` number of filtersearches?

**Review by b1lly — DISMISSED** [review]
Otherwise LGTM

File: README.md (hunk 1 above)
Comment (Delerme) [reply README.md:247]: Yea definitely worth mentioning, done.

**Review by b1lly — DISMISSED** [review]

**Comment by Delerme** [comment]
rebased

**Review by b1lly — APPROVED** [review]

**Pull request merged**